src/test/java/subway/create/LineCreator.java

```diff
@@ -0,0 +1,78 @@
+package subway.create;
+
+import org.springframework.http.MediaType;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import subway.fixture.LineRequestFixture;
+import subway.line.LineCreateRequest;
+import subway.line.LineResponse;
+
+public class LineCreator {
+	private final String upStationName;
+	private final String downStationName;
+
+	private LineCreator(String upStationName, String downStationName) {
+		this.upStationName = upStationName;
+		this.downStationName = downStationName;
+	}
+
+	public static Builder builder() {
+		return new Builder();
+	}
+
+	public static Builder init() {
+		return LineCreator.builder()
+			.upStationName("강남역")
+			.downStationName("양재역");
+	}
+
+	private ExtractableResponse<Response> action(LineCreateRequest request) {
+		return RestAssured
+			.given().log().all()
+			.body(request)
+			.contentType(MediaType.APPLICATION_JSON_VALUE)
+			.when().post("/lines")
+			.then().log().all()
+			.extract();
```

Comment:
사전 데이터 생성 시 실제로 생성되었는지 확인 없이 진행할 경우 의도하지 않는 결과를 마지할 수 있습니다.
이를 방지하는 최소한의 방법으로 응답 코드 확인이 있을 것 같네요!
```suggestion
			.statusCode(201).extract();
```

Reply:
그렇군요 그럴 경우도 있겠네요!
다음 단계 pr에서 수정해 놓도록 하겠습니다~!